docs/content/docs/try-flink-kubernetes-operator/quick-start.md

```diff
@@ -63,22 +63,15 @@ minikube start --kubernetes-version=v1.21.5
 
 We also recommend [k9s](https://k9scli.io/) as GUI for kubernetes, but it is optional for this quickstart guide.
 
-## Building the operator image from source
+## Get the source code
+
+You can get the operator source code from GitHub:
 
 ```shell
 git clone https://github.com/apache/flink-kubernetes-operator.git
 cd flink-kubernetes-operator
 ```
 
-You can build the docker image of the operator via the following command from the project root folder:
```

Comment:
Why would you want to get the source code if you are not going to build the operator?